src/applications/claims-status/sass/claims-status.scss

```diff
@@ -556,6 +556,16 @@ h1:focus {
 .claim-timeline {
   padding-top: 0;
 
+  h5 button {
+    all: inherit;
```

Comment:
This isn't supported in IE or Edge, looks like: https://caniuse.com/#feat=css-all

Reply:
I don't see any `reset` helpers/mixins in formation -- @jbalboni are you aware of any such helper class we can leverage here? Also a bit of Googling shows no clear results to polyfill this particular feature. . Else I'll rewire the css to be more explicit. Good catch!

Reply:
I made a mixin for the va-button-link class that might work? As a starting point, at least.